package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "smartling-api-sdk-nodejs",
-  "version": "2.16.1",
+  "version": "2.16.2",
```

Comment:
You've added only new features with no BC breaks, sounds like a minor version update, like `2.17.0`. It's not bug fixes for patch version update.